modules/apps/segments/segments-web/src/main/resources/META-INF/resources/css/_criteria_row.scss

```diff
@@ -33,6 +33,11 @@
 		.criterion-input {
 			flex: 1;
 			margin-right: 8px;
+			min-width: 8em;
```

Comment:
We probably would want to stick with `px` for width unless you purposely want a ratio from the parent element.